cmd/brew-brewdle.rb

```diff
@@ -9,15 +9,21 @@
 
 usage = <<-EOS.undent
   brew brewdle
+  brew brewdle dump [--force] [--gen-cleanup-installed]
   brew brewdle [--version]
   brew brewdle [-h|--help]
 
   Usage:
   Bundler for non-Ruby dependencies from Homebrew
 
+  brew brewdle                    read Brewfile and install all dependencies
+  brew brewdle dump               dump current installation into Brewfile
+
   Options:
-  -h, --help        show this help message and exit
-  --version         show the version of brewdler
+  --force                        force overwrite existed Brewfile
+  --gen-cleanup-installed        also generate `.brew-cleanup-installed` file
```

Comment:
Let's pass on this; that could be pulled into that tool instead and/or we could migrate that tool to uninstall everything not in a `Brewfile`